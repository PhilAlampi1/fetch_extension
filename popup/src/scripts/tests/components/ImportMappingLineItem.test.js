import React from 'react'
import { shallow } from 'enzyme'
import ImportMappingLineItem from '../../components/ImportMappingLineItem'

test('ImportMappingLineItem should render correctly with input values', () => {
    const handleOnChangeSpy = jest.fn()
    const action = shallow(<ImportMappingLineItem 
        standardFieldId="123"
        importedFieldName="# Bedrooms"
        handleOnChange={handleOnChangeSpy}
        standardFieldName="Num Bedrooms"
        standardFieldDescription="Number of bedrooms in the property."
        />)
    expect(action).toMatchSnapshot()
})

test('ImportMappingLineItem should render correctly with input values', () => {

})