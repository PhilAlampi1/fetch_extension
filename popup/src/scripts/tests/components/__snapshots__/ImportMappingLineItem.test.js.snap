// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportMappingLineItem should render correctly with input values 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ImportMappingLineItem
    handleOnChange={[Function]}
    importedFieldName="# Bedrooms"
    standardFieldDescription="Number of bedrooms in the property."
    standardFieldId="123"
    standardFieldName="Num Bedrooms"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(SelectWithImportOptions)
          handleOnChange={[Function]}
          importedFieldName="# Bedrooms"
          standardFieldId="123"
        />,
        <span>
          Num Bedrooms
        </span>,
        <span>
          Number of bedrooms in the property.
        </span>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "handleOnChange": [Function],
          "importedFieldName": "# Bedrooms",
          "standardFieldId": "123",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Num Bedrooms",
        },
        "ref": null,
        "rendered": "Num Bedrooms",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Number of bedrooms in the property.",
        },
        "ref": null,
        "rendered": "Number of bedrooms in the property.",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(SelectWithImportOptions)
            handleOnChange={[Function]}
            importedFieldName="# Bedrooms"
            standardFieldId="123"
          />,
          <span>
            Num Bedrooms
          </span>,
          <span>
            Number of bedrooms in the property.
          </span>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "class",
          "props": Object {
            "handleOnChange": [Function],
            "importedFieldName": "# Bedrooms",
            "standardFieldId": "123",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Num Bedrooms",
          },
          "ref": null,
          "rendered": "Num Bedrooms",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Number of bedrooms in the property.",
          },
          "ref": null,
          "rendered": "Number of bedrooms in the property.",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
